verilog/tools/ls/verilog-language-server_test.cc

```diff
@@ -593,6 +593,13 @@ std::string ReferencesRequest(absl::string_view file, int id, int line,
                                           line, character);
 }
 
+// Creates a textDocument/hover request
+std::string HoverRequest(absl::string_view file, int id, int line,
```

Comment:
is the `id` actually a relevant value we need to pass for the tests ? If not maybe we just replace it with some `static constexpr int kDummyId = 1;` to not distract from the other values.

Reply:
It should be unique per request - we can go with the static value that is incremented per request

Reply:
Yep, but is any place even checking if it is unique per request ?